charts/connect/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "redpanda-connect.fullname" . }}-test-connection"
+  labels:
+    {{- include "redpanda-connect.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
```

Comment:
Could you clone the GHA setup for the other charts? Make a `.github/workflows/test_<chart-name>.yaml` and then connect it via `.github/workflows/pull_request.yaml`.

No need to setup chart testing or any of that, running `helm test` on a kind cluster for now would be fine IMO.